content/kots-cli/upload.md

```diff
@@ -5,4 +5,26 @@ title: kots upload
 weight: 90140
 ---
 
-If you have a copy of an application that was created with kots pull or kots download, you can upload it back to the Admin Console using the `kots upload` command.
+Upload Kubernetes manifests from the local filesystem to a cluster, creating a new version of the application that can be deployed. When you have a copy of an application that was created with `kots pull` or `kots download`, you can upload it back to the Admin Console using the `kots upload` command. 
```

Comment:
~to a cluster~ to the Admin Console?  I wonder if saying that it uploads to the cluster could be interpreted as a way of saying that the application is deployed?  This command just makes it available as a new version in the Admin Console.

Reply:
Thanks. Will adjust wording.

Reply:
Note that the wording from "--help" is presently "Upload Kubernetes manifests from the local filesystem to a cluster, creating a new version of the application that can be deployed." So we probably want to update that as well.